assets/js/downloads.js

```diff
@@ -44,21 +44,22 @@ jQuery(document).ready(function ($) {
       url: Tari.s3BucketURL,
       headers: { "Access-Control-Allow-Origin": "*" },
       success: function (res) {
-        groupDataByOs(res.current);
-        setLatest(res.latest);
+        groupDataByOs(res);
+        setLatest(res);
       },
     });
   }
 
   function groupDataByOs(data) {
-    renderBinaries(data.linux, "linux");
-    renderBinaries(data.windows, "windows");
-    renderBinaries(data.osx, "osx");
-    renderBinaries(data.libwallet, "libwallet");
+    renderBinaries(data["current/linux"], "linux");
+    renderBinaries(data["current/windows"], "windows");
+    renderBinaries(data["current/osx"], "osx");
+    renderBinaries(data["current/libwallet"], "libwallet");
```

Comment:
The dom is expecting `libWalletBinID`
```suggestion
    renderBinaries(data["current/libwallet"], "libWallet");
```